### ISSUE TITLE
ZoneHVAC:WaterToAirHeatPump inlet/outlet flow mismatch during supplemental heat operation

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5430]
Followup from #5422 
Example file ZoneVSWSHP_wDOAS has unbalanced exhaust flow warning. If you change to detailed output and report the mass flow rate at all nodes, there are two timesteps (running with Chicago weather) where the supplemental heat kicks on at 7am and the ZoneHVAC:WaterToAirHeatPump zone inlet mass flow rate is 1.42 while the outlet flow rate (into the WSHP) is 2.10. The WSHP mixed air node also shows the higher flow rate.

**Comment by rraustad** [comment]
Sizing issue where the fan limits flow when HVAC system exceeds max fan flow:

```
Component Sizing Information, ZoneHVAC:WaterToAirHeatPump, ZONE1WTAHP, Design Size Cooling Supply Air Flow Rate [m3/s], 1.21155
Component Sizing Information, ZoneHVAC:WaterToAirHeatPump, ZONE1WTAHP, Design Size Heating Supply Air Flow Rate [m3/s], 1.21155

Component Sizing Information, COIL:COOLING:WATERTOAIRHEATPUMP:VARIABLESPEEDEQUATIONFIT, SYS 1 HEAT PUMP COOLING MODE, Design Size Rated Air Flow Rate [m3/s], 2.42639
Component Sizing Information, COIL:HEATING:WATERTOAIRHEATPUMP:VARIABLESPEEDEQUATIONFIT, SYS 1 HEAT PUMP HEATING MODE, Design Size Rated Air Flow Rate [m3/s], 1.81928

Component Sizing Information, Fan:OnOff, ZONE 1 FAN, Design Size Maximum Flow Rate [m3/s], 1.21155
```

**Comment by rraustad** [comment]
This is ugly. Moving conversation to https://github.com/NREL/EnergyPlus/pull/5632

**Comment by mjwitte** [comment]
Closed via #5632
